evap/evaluation/models.py

```diff
@@ -958,6 +958,22 @@ def annotate_with_participant_and_voter_counts(cls, evaluation_query):
             num_voters=Subquery(voter_count_subquery),
         )
 
+    @classmethod
+    def get_sorted_evaluation_url_tuples_with_urgent_review(cls) -> List[Tuple["Evaluation", str]]:
+        evaluation_url_tuples = [
+            (
+                evaluation,
+                reverse(
```

Comment:
hmm, sorry, I should have been more clear here: `reverse` only gives the path component of the URL, but not the hostname (i.e. this gives `/semester/1/evaluation/1/textanswers`, without `https://example.com`). So we still need to manually add the host name.

The tests currently pass, so we should probably add a test for the URL generated here.